demo/src/main/java/com/example/demo/Tournament.java

```diff
@@ -0,0 +1,59 @@
+package com.example.demo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+public class Tournament {
+    private static  List<Team> winners = new ArrayList<>();
+    private static Random random = new Random();
+    private static Scanner scanner = new Scanner(System.in);
```

Comment:
Consider extracting logic that operates with console into separate component. This class has too many responsibilities